lambda_src/drivers/process_https.py

```diff
@@ -136,12 +137,12 @@ def process_row(
     req_results_path: str = results_path
     req_cursor: str = cursor
     req_method: str = method.upper()
-
     if json:
         req_data: Optional[bytes] = (
-            json if json.startswith('{') else dumps(parse_header_dict(json))
+            json
+            if (isinstance(json, str) or json.startswith('{'))
```

Comment:
I think you mean to say `and` not `or` here. also please use pyblack or manually remove the extra parentheses